include/functions.h

```diff
@@ -3445,9 +3418,9 @@ UNK_TYPE func_801A51F0(UNK_TYPE arg0);
 // void func_801A5808(void);
 // void func_801A5A10(void);
 // void func_801A5A1C(void);
-void func_801A5BD0(s32 param_1);
-// void func_801A5C28(void);
-// void func_801A5C8C(void);
+void Audio_SetSfxBanksMute(u16 muteMask);
+void Audio_SetFlagForBgmVolumeLow(u8 arg0);
+void Audio_ClearFlagForBgmVolumeLow(u8 arg0);
 void Audio_PlaySfxGeneral(u16 sfxId, Vec3f* param_2, u8 param_3, f32* param_4, f32* param_5, s8* param_6);
```

Comment:
Actually same here, these are very obscure at the moment!

Reply:
Are you referring to `Audio_PlaySfxGeneral` specifically? The args are named in https://github.com/zeldaret/mm/pull/798/files. But I'll update the other args as those are being changed here